spec/board_spec.rb

```diff
@@ -0,0 +1,62 @@
+require './lib/board' 
+
+RSpec.describe Board do
+  describe "initialize" do
+    it "returns an array with nil cells value" do
+      board = Board.new(3)
+      expect(board.matrix).to eql([nil,nil,nil,nil,nil,nil,nil,nil,nil])
+    end
+    it "returns an array with combinaisons win value" do
+      board = Board.new(3)
+      expect(board.win_combinations).to eql([[0, 1, 2],[3, 4, 5],[6, 7, 8],[0, 3, 6],[1, 4, 7],
+      	[2, 5, 8],[0, 4, 8],[2, 4, 6]])
```

Comment:
Again, I don't think you need to test for this. This is more related to the inner workings of your class